去掉INSTALL那行编译, 会有报错:
```
[100%] Linking CXX static library ../output/lib/libbraft.a
[100%] Built target braft-static
[ 89%] Performing install step for 'extern_braft'
[ 91%] Built target OBJ_LIB
[ 93%] Built target braft-static
[ 95%] Linking CXX shared library ../output/lib/libbraft.dylib
[ 97%] Building CXX object tools/CMakeFiles/braft_cli.dir/braft_cli.cpp.o
Undefined symbols for architecture arm64:
  "GetStackTrace(void**, int, int)", referenced from:
      butil::debug::EnableInProcessStackDumping() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::EnableInProcessStackDumping() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::(anonymous namespace)::StackDumpSignalHandler(int, __siginfo*, void*) in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::(anonymous namespace)::StackDumpSignalHandler(int, __siginfo*, void*) in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::StackTrace::StackTrace() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::StackTrace::StackTrace() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::StackTrace::StackTrace() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::StackTrace::StackTrace() in libbrpc.a[29](stack_trace_posix.cc.o)
      ...
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[5]: *** [output/lib/libbraft.dylib] Error 1
make[4]: *** [src/CMakeFiles/braft-shared.dir/all] Error 2
make[4]: *** Waiting for unfinished jobs....
[100%] Linking CXX executable output/bin/braft_cli
Undefined symbols for architecture arm64:
  "GetStackTrace(void**, int, int)", referenced from:
      butil::debug::EnableInProcessStackDumping() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::EnableInProcessStackDumping() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::(anonymous namespace)::StackDumpSignalHandler(int, __siginfo*, void*) in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::(anonymous namespace)::StackDumpSignalHandler(int, __siginfo*, void*) in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::StackTrace::StackTrace() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::StackTrace::StackTrace() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::StackTrace::StackTrace() in libbrpc.a[29](stack_trace_posix.cc.o)
      butil::debug::StackTrace::StackTrace() in libbrpc.a[29](stack_trace_posix.cc.o)
      ...
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[5]: *** [tools/output/bin/braft_cli] Error 1
make[4]: *** [tools/CMakeFiles/braft_cli.dir/all] Error 2
make[3]: *** [all] Error 2
make[2]: *** [third-party/braft/src/extern_braft-stamp/extern_braft-install] Error 2
make[1]: *** [CMakeFiles/extern_braft.dir/all] Error 2
make: *** [all] Error 2
 pikiwidb compile fail
```

--- a/cmake/brpc.cmake
+++ b/cmake/brpc.cmake
@@ -41,7 +41,6 @@ EXTERNALPROJECT_ADD(
         -DCMAKE_BUILD_TYPE:STRING=${THIRD_PARTY_BUILD_TYPE}
         BUILD_IN_SOURCE 1
         BUILD_COMMAND $(MAKE) -j ${CPU_CORE} brpc-static
-        INSTALL_COMMAND mkdir -p ${BRPC_INSTALL_DIR}/lib/ COMMAND cp ${BRPC_SOURCES_DIR}/src/extern_brpc/output/lib/libbrpc.a ${BRPC_LIBRARIES} COMMAND rm -rf ${BRPC_INCLUDE_DIR} COMMAND cp -r ${BRPC_SOURCES_DIR}/src/extern_brpc/output/include ${BRPC_INCLUDE_DIR}
 )
 ADD_DEPENDENCIES(extern_brpc ssl crypto zlib protobuf leveldb gflags)
 ADD_LIBRARY(brpc STATIC IMPORTED GLOBAL)

